stable/kommander/charts/kubecost/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: 1.0.0
```

Comment:
should this be `1.55.0`? based on [this](https://github.com/mesosphere/charts/pull/523/files#diff-5de584009d684fa30ea6c164a931f218R73)